src/com_tjvendors/admin/sql/install.mysql.utf8.sql

```diff
@@ -38,6 +38,16 @@ CREATE TABLE IF NOT EXISTS `#__tjvendors_vendors` (
   `vendor_id` int(11) NOT NULL AUTO_INCREMENT,
   `user_id` int(11) NOT NULL,
   `vendor_title` varchar(255) NOT NULL,
+  `first_name` varchar(255) NOT NULL,
+  `last_name` varchar(255) NOT NULL,
+  `address` text NOT NULL,
```

Comment:
@deepa-g @manojLondhe Can you use Joomla Custom fields?

Reply:
This MR will close